meta-signing-key/recipes-support/key-store/key-store_0.1.bb

```diff
@@ -24,7 +24,7 @@ SECONDARY_TRUSTED_PRIV_KEY = "${KEY_DIR}/secondary_trusted_key.key"
 MODSIGN_PRIV_KEY = "${KEY_DIR}/modsign_key.key"
 
 # For ${PN}-ima-privkey
-IMA_PRIV_KEY = "${KEY_DIR}/privkey_evm.crt"
```

Comment:
This file name matches up the default key name used by the signing tool provided by ima-evm-utils.